### DISCUSSION TITLE
Whats is the bwet way to use i18next with next for SSR?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#13514] [Help]
Hi guys!

Is there an official way to use i18next or another translation lib? (I have all languages ​​in a key / value table).

Is there an official way to do this? Or are there plans to have an official way in the future? What is the best way.

I'm using Next for SSR.

Thans so much?

**Comment by Timer** [comment]
We are landing a `with-next-translate` example, check it out!
https://github.com/vercel/next.js/pull/13511
